admin_servers/admin_servers_views.py

```diff
@@ -75,6 +84,18 @@ def node_list(request):
             return Response(serializer.data,status=status.HTTP_201_CREATED)
         else:
             return Response(serializer.errors,status=status.HTTP_400_BAD_REQUEST)
+@api_view(['DELETE'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def delete_node(request,url):
```

Comment:
Do you want to put `url` in the DELETE url? It will be something like `www.i-connect.com/service/admin/deletenode/www.group.com/`

Reply:
The best way might DELETE request with a body containing the node object 
```
{
url: xxx, 
user_name: xxx, 
password: xxx
}
```

Reply:
sure thanks